custom/icds_reports/migrations/sql_templates/database_functions/aggregate_awc_data.sql

```diff
@@ -258,7 +258,7 @@ BEGIN
      'num_launched_awcs = 1 ' ||
     'FROM (SELECT DISTINCT(awc_id) ' ||
        'FROM agg_awc ' ||
-  'WHERE month = ' || quote_literal(_previous_month_date) || ' AND usage_num_hh_reg > 0 AND awc_id <> ' || quote_literal(_all_text) || ') ut ' ||
+  'WHERE month = ' || quote_literal(_previous_month_date) || ' AND num_launched_awcs > 0 AND awc_id <> ' || quote_literal(_all_text) || ') ut ' ||
```

Comment:
Might be a more efficient query to also add aggregation_level = 5 so that it can filter out the other tables.